content/en/watchdog/insights.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Watchdog Insights
+kind: documentation
+description: View anomalies and outliers that match your search query with Watchdog Insights
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-watchdog-insights-log-management/"
+  tag: "Blog"
+  text: "Augmented troubleshooting with Watchdog Insights"
+- link: "https://www.datadoghq.com/blog/watchdog-insights-apm/"
+  tag: "Blog"
+  text: "Automatically detect error and latency patterns with Watchdog Insights for APM"
+---
+
+## Overview
+
+Datadog Watchdog constantly runs in the background, scanning for anomalies in your organization's entire data set. As you navigate the Datadog UI, Watchdog Insights displays a filtered and sorted list of anomalies matching your active search query.
+
+### Usage
+Investigating an incident requires trial and error. Drawing from their experience, engineers familiar with a particular area know where to first look for potential problems. Using Watchdog Insights allows all engineers, including less experienced ones, to pay attention to the most important data and accelerate their incident investigations.
+
+Each insight highlights one outlier or anomaly affecting a subset of users. Depending on the product area, Watchdog Insights displays different types of anomalies. Examples include, but are not limited to, the following:
+- Error and latency outliers in logs, traces, and RUM views
+- Spike in error logs
+- New error logs
+- Deadlocked threads
+- High percentage of unready Kubernetes pods
+
+### Prioritization
+
+Watchdog sorts insights based on a combination of factors to place the most important insight at the beginning of the list. The factors that Watchdog takes into account can include the following:
+- State (ongoing versus resolved)
+- Status (warning, error, or critical)
+- Start time
+- Anomaly type
+
+## Navigation
+
+{{< img src="watchdog/log_explorer_watchdog_insights.png" alt="The Watchdog Insights banner on the Logs Explorer, showing three anomalies: new error logs in the web-store service, a spike in error logs in the product-recommendation service, and another spike in error logs in the product-recommendation service" >}}
+
+The Watchdog Insights banner sits near the top of each page. Expand the banner for an overview. The highest priority insights appear on the left. If Watchdog cannot find any issues, the banner is gray.
+
+### Filter on Insight
+
+To refine your current view to match a Watchdog Insight, hover over the top right corner of an insight summary card. Two icons appear. Click on the inverted triangle icon with the tooltip **Filter on Insight**. The page refreshes to show a list of entries corresponding to the insight.
+
+### Side panel
+
+Click **View all** to expand the panel. A side panel opens from the right, containing a vertical list of Watchdog Insights. Each entry shows a detailed view, with more information than the summary card.
+
```

Comment:
This would benefit from a "View All" side panel screenshot (eg. from Log Management): these side panels show neatly a detailed view of multiple Insights are already very actionable, even before users click on a specific Insight card